tests/test_order_SolidLiquid.py

```diff
@@ -35,21 +35,27 @@ def test_nlist(self):
 
         npt.assert_array_equal(nlist[:], comp.nlist[:])
 
-    def test_identical_environments(self):
+    comp_default = freud.order.SolidLiquid(6, q_threshold=0.7, solid_threshold=6)
+    comp_no_norm = freud.order.SolidLiquid(
+        6, q_threshold=0.3, solid_threshold=6, normalize_q=False
+    )
+
+    @pytest.mark.parametrize(
+        "comp, query_args",
+        [
+            (comp, query_args)
+            for comp in (comp_default, comp_no_norm)
+            for query_args in (dict(r_max=2.0), dict(num_neighbors=12))
+        ],
+    )
+    def test_identical_environments(self, comp, query_args):
```

Comment:
This test is another case like `GaussianDensity` where by adding the parametrization we have implicitly lost test coverage for calling compute multiple times in a row. Please add a small test which covers this case.